src/control/server/superblock.go

```diff
@@ -100,7 +100,11 @@ func (srv *IOServerInstance) hasSuperblock() bool {
 
 // NeedsSuperblock indicates whether or not the instance appears
 // to need a superblock to be created in order to start.
-func (srv *IOServerInstance) NeedsSuperblock() (bool, error) {
+func (srv *IOServerInstance) NeedsSuperblock(needsScmFormat bool) (bool, error) {
+	if needsScmFormat {
+		return true, errors.New("can't read superblock from unformatted storage")
+	}
+
```

Comment:
This seems a little odd. Wouldn't it be better to just not call this method if we know the value of `needsScmFormat` and it's true?

Reply:
yes I can do that, I just preferred to have the parameter as a sanity check